ophyd/__init__.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import types
 import os
+from ophyd log import set_handler
```

Comment:
This is a `SyntaxError`.

Reply:
Sorry, I forgot to make the final commit before opening the PR.